healthcheck.js

```diff
@@ -0,0 +1,40 @@
+const http = require('http')
+
+const availableClients = {
+  all: '',
+  zwave: '/zwave',
+  mqtt: '/mqtt'
+}
+
+let clientCheck = availableClients.all
+
+if (process.env.HEALTH_CLIENT) {
+  for (const c in availableClients) {
+    if (c === process.env.HEALTH_CLIENT) {
+      clientCheck = availableClients[c]
+      break
+    }
+  }
+}
+
+const options = {
+  port: require('./config/app.js').port,
+  timeout: 2000,
```

Comment:
you've a 2 sec timeout here but a 12sec timeout in the healthcheck itself?

https://github.com/zwave-js/zwavejs2mqtt/pull/1211/files#diff-f34da55ca08f1a30591d8b0b3e885bcc678537b2a9a4aadea4f190806b374ddcR66

Reply:
In case the healthcheck stucks for any other reason I dind't think about 😆 But I could set the two timeouts to be equal, what do you think it's best?